src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect } from 'react';
+import {
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { METRIC_TYPE } from '@kbn/analytics';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { KibanaPageTemplate } from '@kbn/shared-ux-components';
+
+import { getServices } from '../../kibana_services';
+import { UseCaseCard } from './use_case_card';
+
+import './getting_started.scss';
+
+export const GettingStarted = () => {
+  const { application, trackUiMetric, chrome } = getServices();
+
+  useEffect(() => {
+    chrome.setBreadcrumbs([
+      {
+        // using # prevents a reloading of the whole app when clicking the breadcrumb
+        href: '#',
+        text: i18n.translate('home.breadcrumbs.homeTitle', { defaultMessage: 'Home' }),
+        // TODO telemetry for navigating away from getting started via breadcrumbs
+        onClick: () => {},
+      },
+      {
+        text: i18n.translate('home.breadcrumbs.gettingStartedTitle', {
+          defaultMessage: 'Getting Started',
```

Comment:
I _think_ we generally follow sentence case
```suggestion
          defaultMessage: 'Getting started',
```

Reply:
That is correct